doc/howtos/rdtraining/10_actions.rst

```diff
@@ -0,0 +1,139 @@
+.. _howto/rdtraining/10_actions:
+
+===============================
+Part 10: Ready For Some Action?
+===============================
+
+Up to now, we mostly built our module by declaring fields and views. We only introduced business
+logic in the :ref:`previous chapter <howto/rdtraining/09_compute_onchange>` thanks to computed fields
+and onchanges. In any real business scenario, we want to link some business logic to action buttons.
+In our real estate example, we would like to be able to:
+
+- cancel or set a property to sold
+- accept or refuse an offer
+
+One could argue that it is already something that we can do by changing the state manually, but
+this is not really convenient. Moreover, we want to add some extra processing: when an offer is
+accepted we want to set the selling price and the buyer on the property.
+
+Action Type
+===========
+
+**Reference**: the documentation related to this topic can be found in
+:ref:`reference/actions` and :ref:`reference/exceptions`.
+
+.. note::
+
+    **Goal**: at the end of this section:
+
+    - You can cancel or set a property to sold:
```

Comment:
```suggestion
    - You can cancel or set a property to sell:
```
saw this one while re-looking at the structure of the main doc.

Reply:
Incidentally rebasing / rewriting this one is super annoying when building branches / PRs for the advanced sections.